Sources/PackageModel/Product.swift

```diff
@@ -89,6 +89,8 @@ public enum ProductFilter: Equatable, Hashable {
     /// A set of specific products requested by one or more client packages.
     case specific(Set<String>)
 
+    public static var nothing: ProductFilter { .specific([]) }
```

Comment:
Perhaps a nitpick but it would be great to add a doc comment here.

Reply:
Not at all! Thanks for the reminder. Added in b86b4096610104dd103392b755515098ca14ed6f.